git-issue.sh

```diff
@@ -1145,14 +1145,22 @@ sub_list()
   esac
   shift $((OPTIND - 1));
   tag="$1"
-  : "${tag:=open}"
+  if [ "$tag" = "closed" ] ; then
+    # when explicitly searching for closed tickets, don't show only open ones
+    all=1
+  fi
   cdissues
   test -d issues || exit 0
   find issues -type f -name tags -o -name milestone |
+  if [ "$tag" ] ; then
+    xargs grep -Flx "$tag"
+  else
+    cat
+  fi |
   if [ "$all" ] ; then
     cat
   else
-    xargs grep -Flx "$tag"
+    sed 's/\/milestone$/\/tags/' - | xargs grep -Flx open
```

Comment:
Why is the *sed* invocation required? If it's for another bug, can you please open a separate PR?

Also please remove the unneeded `-` from the *sed* invocation.

Reply:
sed is necessary because the `find issues -type f -name tags -o -name milestone` can return either the tags file or the milestone file, if it's the milestone file I use sed to change it to tags so I can check for the open tag there.

I removed the -